### ISSUE TITLE
Excluding posts from feeds

### ISSUE THREAD

**Issue by threegreenbirds** [danohart/threegreenbirds#8]
Could you make it so that any post tagged with "exclude-from-feeds" is not included in the RSS feed for the site?

**Comment by danohart** [comment]
Yes, I sure can. Will get on that now and will let you know when it is implemented.

**Comment by danohart** [comment]
Just made this change. So now when you create or edit a blog post, you can check the category "Exclude from Feeds", and it will not show up in RSS feeds.

**Comment by threegreenbirds** [comment]
It looks like you implemented this using a Category term, when the request was to use a tag. Could you refactor accordingly? Also, could you make sure the tag's ID is not hardcoded, using the slug instead? Lastly, could you point to some example URLs that demonstrate the implementation once you have refactored? Thanks.
